icekit/authors/tests.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+from django_dynamic_fixture import G
+from django_webtest import WebTest
+from fluent_contents.models import Placeholder
+
+from . import models
+
+User = get_user_model()
+
+class AuthorTests(WebTest):
+    def setUp(self):
+        self.staff_1 = User.objects.create(
+            email='test@test.com',
+            is_staff=True,
+            is_active=True,
+            is_superuser=True,
+        )
+
+        self.author_listing = models.AuthorListing.objects.create(
+            author=self.staff_1,
+            title="Authors",
+            slug="authors",
+        )
```

Comment:
Maybe add a comment "We need a listing page so `get_absolute_url()` works on detail objects" or something?